app/models/preneeds_attachment_type.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+require 'common/models/base'
+
+# DischargeType model
+class PreneedsAttachmentType < Common::Base
+  include ActiveModel::Validations
+
+  validates :description, :attachment_type_id, presence: true
```

Comment:
for attachments I think we will likely need additional validations such as individual file size, total file size of all attachments, maximum number of attachments, but perhaps those validations would go on the attachments aggregate (somewhere on the form object itself).